tests/_utils/start_tidb_cluster

```diff
@@ -123,4 +123,4 @@ while ! mysql -uroot -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} --default-character
 done
 
 run_sql "update mysql.tidb set variable_value='60m' where variable_name='tikv_gc_life_time';"
-run_sql_down "update mysql.tidb set variable_value='60m' where variable_name='tikv_gc_life_time';"
+run_sql "update mysql.tidb set variable_value='60m' where variable_name='tikv_gc_life_time';" 127.0.0.1 3306
```

Comment:
Can we use variables like `$DOWN_TIDB_HOST` here?

Reply:
yes, fixed